longevity_mv_synchronous_updates_test.py

```diff
@@ -0,0 +1,63 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2020 ScyllaDB
+
+# This is stress longevity test that runs light weight transactions in parallel with different node operations:
+# disruptive and not disruptive
+#
+# After the test is finished will be performed the data validation.
+
+from longevity_test import LongevityTest
+from sdcm.sct_events.database import DatabaseLogEvent
+from sdcm.sct_events.filters import DbEventsFilter
+from sdcm.utils.common import keyspace_min_max_tokens
+from sdcm.sct_events.group_common_events import ignore_mutation_write_errors
+
+
+class MvSynchronousLongevityTest(LongevityTest):
+
+    def run_prepare_write_cmd(self):
+        with ignore_mutation_write_errors():
+            super().run_prepare_write_cmd()
+
+        self.log.debug("Start create_main_mv")
+        self.create_main_mv()
+        self.log.debug("Finish create_main_mv")
+
+    def test_mv_synch_updates_longevity(self):
+        with DbEventsFilter(db_event=DatabaseLogEvent.WARNING, line="perf_event_open"):
```

Comment:
what is this filter ? "perf_event_open" why do we need it for this specific case ?

Reply:
removed